scripts/run_tests_unit.sh

```diff
@@ -4,9 +4,15 @@
 #
 # NOTE: This script expects to be run from the project root with
 # ./scripts/run_tests_unit.sh
+current_dir=$(dirname "${BASH_SOURCE[0]}")
+parent_dir=$(dirname "${current_dir}")
+path_to_parent="$( cd "${parent_dir}" && pwd )"
+temp_dir="${path_to_parent}"/temp
 
 set -o pipefail
 
+curl 'https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.json' --compressed --create-dirs -o "${temp_dir}"/blns.json
```

Comment:
Ok, I've added a fixtures folder under the tests directory and added committed the json file. Seem sensible?